### PR TITLE
examples: move TLS example to a separate project

### PR DESCRIPTION
**Pull request by dapengzhang0**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,6 +11,8 @@ target
 # Bazel
 bazel-bin
 bazel-examples
+bazel-examples-alts
```
Comment (ejona86) [inline .gitignore]: No, let's not create a separate workspace for each set of examples.

File: examples/build.gradle [hunk 2]
```diff
@@ -23,23 +23,19 @@ targetCompatibility = 1.7
...
-    implementation "io.grpc:grpc-netty-shaded:${grpcVersion}"
     implementation "io.grpc:grpc-protobuf:${grpcVersion}"
     implementation "io.grpc:grpc-stub:${grpcVersion}"
     compileOnly "javax.annotation:javax.annotation-api:1.2"
 
-    // Used in HelloWorldServerTls
-    implementation "io.grpc:grpc-netty:${grpcVersion}"
-    implementation "io.netty:netty-tcnative-boringssl-static:${nettyTcNativeVersion}"
-
+    // examples/advanced need this for JsonFromat
```
Comment (ejona86) [inline examples/build.gradle]: s/Fromat/Format/

File: .gitignore (hunk 1 above)
Comment (dapengzhang0) [reply .gitignore]: Done.

File: examples/build.gradle (hunk 2 above)
Comment (dapengzhang0) [reply examples/build.gradle]: Done.

**Pull request merged**